src/azure-cli/azure/cli/command_modules/containerapp/tests/latest/test_containerapp_commands.py

```diff
@@ -618,6 +618,20 @@ def test_containerapp_cors_policy(self, resource_group):
             JMESPathCheck('maxAge', 0),
         ])
 
+        self.cmd(
+            'containerapp ingress cors enable -g {} -n {} --allowed-origins "*"  --allow-credentials True --max-age "" '.format(
```

Comment:
should you add test for the scenario reported

Reply:
Already added test for `az containerapp ingress cors ` without --allow-credencial in line 568

Reply:
![image](https://github.com/Azure/azure-cli/assets/46143499/e0908b5c-6cd5-4e9d-ba5e-29f521572563)